script/Ciel/Day 9/09_01_CIEL09_4.txt

```diff
@@ -381,42 +451,58 @@ From what I heard, the longest time she's ever taken was one year, during which
+That's why she chose to infiltrate the city through a culvert and hide herself in the sewers.
+ Those pipes may have been like blood vessels to the Dead Apostle,
+ but his senses didn't extend to the filth that drifted through those veins."
 }
 [sha:8dccb5c60c32dd755d24a426edbb2338c6940e0e]{
 -- Page 406, Offset 23180.
 -- 「秘蹟も魔術も使えば感知される状況だったから、あの子は黒鍵の<柄|え>を胃に呑み込んで下水に入っていった。
-"Since using a Sacrament or magecraft would make her presence known, she swallowed the handle of a Black Key before entering the sewers.
+// Parsing 柄 as just the handle doesn't make much sense here - assuming it's more like the stalk/length of an _entire_ black key?
+// Hard to do much stabbing with just the hilt, surely
+"Since using any sacraments or magecraft would give away her presence,
+ she swallowed the entire length of a black key before entering the sewers.
 }
 [sha:64539c7ba6d7e208e70ec1711a5d9a028cf7a33a]{
 -- Page 406, Offset 23181.
 -- この国の整備された下水じゃない。汚物でつまったヨーロッパの下水路で、体を汚水にひたして、顔だけを水面に出してギリギリの呼吸をして、ただ機会を待ち続ける一年間！」
-Those sewers weren't like this country's well-maintained drainage systems. She bode her time for a whole year, barely able to breathe. Only her head was above the water, while the rest of her body was submerged in the black water of some filthy European sewer!"
+// Wonder if she came into contact with the fatberg (rip)
+Those sewers weren't as well-maintained as the ones in this country.
+ She spent an entire year biding her time with her body immersed in the effluent of some European sewer,
+ only her face above water so that she could breathe,
+ waiting for her chance to strike."
 }
 [sha:d4038b362d003da3d8c5577eaf9c78db7dc33a66]{
 -- Page 407, Offset 23182.
 -- 「……ほんと、想像するだけで寒気がする。この件ばっかりは吸血鬼に同情するわ。
-"...Seriously, just imagining it makes me shudder. I feel bad for the vampire.
+"Ugh, just imagining it makes me shudder.
+ Just this once, I actually feel sorry for the vampire.
 }
 [sha:ac571c40cac1b45f658947d5fb0b2dd225bcbccd]{
 -- Page 407, Offset 23183.
 -- 街を完全に支配下において、やってきた代行者も全員殺して、ようやく自由に闊歩できると安心しきった死徒が広場に踏み出した瞬間、下水口の網の下から一撃で即死よ？　蛇だってあの子の執念深さには尻尾を巻くわ」
-The Dead Apostle put the city under his complete control, and killed all the Executors that came. He could finally rest easy and wander about freely, but the moment he stepped into the open ground, he was killed by a single strike that came from beneath a sewer grate. Seriously, even a snake would turn tail when faced with her tenaciousness."
+The Dead Apostle had taken complete control over the city,
+ and killed each and every one of the Executors that had come after him.
+ They thought they were finally free to roam the city at their leisure,
+ but then the instant they stepped out into the open for the first time they were killed by a single deadly strike that came out of a sewer grate.
+ Even a snake would turn tail and flee at the sight of that woman's vindictiveness."
```

Comment:
Feels way too run-on to me, and it's not really a situation where I think that heightens the delivery much. 
Are snakes vindictive irl? I have no idea either. It's not really their 'character' in en so much as deceptive and backstabbing, but I get the feeling they probably don't exhibit that much in the wild either.

Reply:
Run-on sentences just means it doesn't have proper conjunctions and comma usage; not that it's simply long. The point of having longer sequences is to build up the tension of the sentence, which is what Noel wants to get across. She's essentially telling a small story. I know building larger sentences is more complex, but I think in this case it's sort of the point.

>The Dead Apostle had taken complete control of the city,
>killed all of the Executors coming after it,
>and then stepped out into the open,
>fully convinced that it could finally roam the city unopposed,
>when a single strike coming out of the sewer grates killed it on the spot.